Hints.py

```diff
@@ -730,6 +736,7 @@ def buildWorldGossipHints(spoiler, world, checkedLocations=None):
 
         if world.shopsanity != "off" and "Progressive Wallet" not in world.item_hints:
             world.item_hints.append("Progressive Wallet")
+        world.named_item_pool = world.item_hints
```

Comment:
Good catch, this was intended to be a copy. `item_hints` is the list that ends up in the settings section of the spoiler. To avoid double-hinting an item, I pop it from the list, but this was also removing it from the spoiler. Making a copy lets me mess with the list without affecting the spoiler.